book/src/integrations/radius.md

```diff
@@ -103,82 +103,26 @@ kanidm service-account credential generate --name admin radius_service_account
 ## Deploying a RADIUS Container
 
 We provide a RADIUS container that has all the needed integrations. This container requires some
-cryptographic material, with the following files being in `/etc/raddb/certs`. (Modifiable in the
+cryptographic material, with the following files mounted in `/data`. (Modifiable in the
 configuration)
 
-| filename | description                                                   |
-| -------- | ------------------------------------------------------------- |
-| ca.pem   | The signing CA of the RADIUS certificate                      |
-| dh.pem   | The output of `openssl dhparam -in ca.pem -out ./dh.pem 2048` |
```

Comment:
What was `dh.pem` used for? Given we can just remove it completely does that mean we never needed it?

Reply:
It was part of EAP auth but FreeRADIUS reported not needing it in startup... supposedly doesn't need it anymore?

This is what was in the logs:

```
tls: Setting DH parameters from /etc/raddb/certs/dh.pem - this is no longer necessary.
tls: You should comment out the 'dh_file' configuration item.
```

Reply:
Well I'll be dammed. As long as it works I'm not fussed (the less setup the better), just surprised.